iuf

```diff
@@ -231,42 +238,14 @@ def get_comment(text_or_list):
     return comment
 
 def process_resume(config):
-    """Pick up where we left off."""
-    comment = get_comment(config.args.get("comment", None))
-    activity = config.args.get("activity")
-    config.stages.set_summary("comment", comment)
-
-    stage_status = config.stages.get_stage_status(activity)
-    unpaused_stage = False
-    next_stage = None
-    last_list_index = len(stage_status) - 1
-    for ctr, stagedict in enumerate(stage_status):
-        stage = stagedict["stage"]
-        run_status = config.stages.run_status(stage)
-        if run_status.lower() == 'paused':
-            # Unpause this stage.
-            unpaused_stage = stage
-            break
-        elif stagedict["succeeded"] is True:
-            last_index = ctr + 1  if ctr + 1  <= last_list_index else ctr
-            next_stage = stage_status[last_index]["stage"]
-
-    start_at = None
-    if unpaused_stage:
-        start_at = unpaused_stage
-    elif next_stage:
-        start_at = next_stage
-    else:
-        print("Couldn't find a successful or paused stage to resume.")
-        sys.exit(1)
+    """Resume an activity"""
 
-    config.stages.start_at = start_at
-    process_install(config)
+    config.activity.resume(config)
 
 
 def process_restart(config):
-    """Remove the files associated with this activity and start over."""
-    comment = get_comment(config.args.get("comment", None))
+    """Restart an activity"""
+    config.activity.restart(config)
 
     # Remove the files in state_dir.
     state_dir = config.args.get("state_dir")
```

Comment:
I think this is a rebase error.  Get rid of everything except `config.activity.restart(config)`